src/hb/ot_layout_gsubgpos.rs

```diff
@@ -276,7 +276,7 @@ impl<'a, 'b> skipping_iterator_t<'a, 'b> {
             self.buf_idx += 1;
             let info = &self.ctx.buffer.info[self.buf_idx];
 
-            match self._match(info) {
+            match self.r#match(info) {
```

Comment:
Can't you use just `.match` here? I thought that's the whole point?

Reply:
<img width="646" alt="image" src="https://github.com/RazrFalcon/rustybuzz/assets/47084093/21275621-3086-41dc-ab86-9419ba1154ef">

Doesn't look like it? Or am I doing something wrong?

> I thought that's the whole point?

I don't know what the point is, I just applied the suggestion you made. :D

Reply:
I thought it would allows us to write `match` directly. But it still needs `#r`. Let's leave `match_` then.